### DISCUSSION TITLE
Works: Monterey 12.3 - iMac Early 2009 - with Jensen Eagle 100ac USB WiFi-adapter

### DISCUSSION THREAD

**Discussion by CHithubber** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#213] [General]
Found this forgotten adapter that only had officially a Windows driver when it was available for purchase but works perfectly after installing "Notarized Wireless USB Big Sur Adapter" + "Wireless USB Big Sur Adapter-V12" with instructions.

(also found this product branded as Tenda U9 when I did some research online)

USB Wifi adapter is showing up in System Report - USB (before installing the driver) as:

Product-ID:	         0xc811
Manufacturer-ID:	0x0bda  (Realtek Semiconductor Corp.)
Version:	                 2.00
Serial nummer:	123456
Speed:	                 Up to 480 Mbit/s
Manufacturer:	         Realtek

Says:
Model U9 printed on the adapter itself for identification.

Running on:
iMac11,1 (late 2009)
Quad-Core Intel Core i7
2,8 GHz

MacOS: Monterey 12.3 installed with Opencore.

**Comment by chris1111** [comment]
Great thanks 👏🏻

**Comment by chris1111** [comment]
I will put in the working testing  list [Done](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/commit/11fd45d6596e873a9531c0adfcad77866d141b60)

**Reply by CHithubber** [reply]
I am quite confident that it works with **Tenda U9** as well (exactly the same specs).

**Comment by EnumSet** [comment]
Not working for Alfa AWUS036AC on M1 Mac OS 12.3.1 . Will there be drivers in the future?
